dev/config_conjur_identity/configure-conjur-identity/dev/dev_default.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import (absolute_import, division, print_function)
```

Comment:
I don't think the contents of this file are ever used from `start.sh` - if it is, then it's a quick confirmation that the env is setup correctly, so I'm no against it being included.

Reply:
yes , You are correct . I removed this file.